docs/source/reference/config.rst

```diff
@@ -87,6 +87,14 @@ Available fields and semantics:
     # Default: false.
     disable_ecc: false
 
+  # Custom policy to be applied to all tasks.
+  #
+  # The policy function to be applied and mutate all tasks, which can be used to
+  # enforce certain policies on all tasks.
+  #
+  # See details in: <TODO: add link to policy docs>
+  policy: my_package.skypilot_policy_fn_v1
```

Comment:
Proposal
- field name: changed to `admin_policy`; to make it more clear. I considered "task_policy" but it's a bit unclear which persona should do it.
- comments/verbal: change everywhere accordingly

Reply:
Changed to `admin_policy` for now.